### PR TITLE
AB#87181 Added paging for users table

### PR DESCRIPTION
**Pull request by Antek012**

### CODE REVIEW

File: src/HE.Investments.Common/Extensions/ListExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+using HE.Investments.Common.Contract;
+
+namespace HE.Investments.Common.Extensions;
+
+public static class ListExtensions
+{
+    public static IList<T> TakePage<T>(this IList<T> source, PagingFilter paging)
```
Comment (Carq) [inline src/HE.Investments.Common/Extensions/ListExtensions.cs:7]: Nice. Please use it in `ApplicationRepository.GetApplicationsWithFundingDetails`

**Review by Carq — APPROVED** [review]

File: src/HE.Investments.Common/Contract/PagedResult.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+namespace HE.Investments.Common.Contract;
```
[the remark this answers is not in the document]
Comment (Antek012) [reply src/HE.Investments.Common/Contract/PagedResult.cs]: changed

File: src/HE.Investments.Common/Extensions/ListExtensions.cs (hunk 1 above)
Comment (Antek012) [reply src/HE.Investments.Common/Extensions/ListExtensions.cs]: changed

File: src/HE.Investments.Common/Contract/PagingFilter.cs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+namespace HE.Investments.Common.Contract;
+
+public record PagingFilter(int PageSize = 10, int PageNumber = 1);
```
[the remark this answers is not in the document]
Comment (Antek012) [reply src/HE.Investments.Common/Contract/PagingFilter.cs]: changed

**Review by jjankowski87 — APPROVED** [review]

**Pull request merged**